editor/editor_file_system.cpp

```diff
@@ -579,8 +579,6 @@ void EditorFileSystem::_scan_new_dir(EditorFileSystemDirectory *p_dir, DirAccess
 
 			if (FileAccess::exists(cd.plus_file(f).plus_file("project.godot"))) // skip if another project inside this
 				continue;
-			if (FileAccess::exists(cd.plus_file(f).plus_file(".gdignore"))) // skip if another project inside this
```

Comment:
The previous logic meant to exclude all folders with a `.gdignore` file, your change seems to only take into account the current directory?